docs/grafana/grafana-dashboard.json

```diff
@@ -580,7 +580,7 @@
       "targets": [
         {
           "exemplar": true,
-          "expr": "sum(devworkspace_fail_total) / sum(devworkspace_started_total) * 100.0",
+          "expr": "sum(increase(devworkspace_fail_total[$__range])) / sum(increase(devworkspace_started_total[$__range])) * 100.0",
```

Comment:
Here is the documentation for `$__range`:  https://grafana.com/docs/grafana/latest/dashboards/variables/add-template-variables/#__range